### PR TITLE
Add styles to webpage

### PR DESCRIPTION
**Pull request by nicolacaze**
This includes CSS file and images.

### CODE REVIEW

File: students/nico/index.html [hunk 1]
```diff
@@ -5,44 +5,62 @@
...
-  <header>
-    <h1>Nicolas Lacaze</h1>
-    <h2>Full stack developer</h2>
+  <header class="header-container">
+    <a href="https://tawebbcn.github.io/cohort0191"><img class="header-container-item" src="img/left-arrow.png" alt="back-arrow"></a>
+    <nav class="header-container-item">
+      <ul class="nav-list-container">
+        <li class="nav-list-items"><a href="#about"/>Me</a></li>
+        <li class="nav-list-items"><a href="#portfolio">Portfolio</a></li>
+        <li class="nav-list-items"><a href="#contact">Contact</a></li>
```
Comment (tawebbcn) [inline students/nico/index.html]: I don't think that you need a class for everything, sometimes remember that you can use the element to style it. For example you can use in the css the class of the navbar and then refer to the li.

File: students/nico/index.html [hunk 2]
```diff
@@ -5,44 +5,62 @@
...
+      </div>
+    </section>
+    <section id="about" class="section-container about-me">
         <h3>About me</h3>
         <p>I am a French guy who speaks English with no strong accent. <br>
-          Yes I like cheese but who does not ? I am an Ironhacker since this morning. <br>
-          I love HTML!
+          Yes I like cheese but who does not ? I am a proud Ironhacker. <br>
+          I love flex-box!
         </p>
```
Comment (tawebbcn) [inline students/nico/index.html:43]: ❤️

File: students/nico/index.html [hunk 3]
```diff
@@ -5,44 +5,62 @@
...
+          <label for="comment">Type your message</label>
+            <textarea id="comment" rows="5" cols="50" placeholder="Leave a message"></textarea> <br>
+            <input class="submit-btn" type="submit" value="Submit">
         </form>
-      </footer>
+      </section>
   </main>
+  <footer>
+    <p>Made with love and no papyrus font.</p>
+  </footer>
```
Comment (tawebbcn) [inline students/nico/index.html:66]: ahaha ❤️

File: students/nico/styles.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+  text-align: center;
+  position: relative;
+  left:33%;
+}
+
+
+/* ---- section ---- */
+.intro {
+  background: url("../myPersoPage/img/georgie-cobbs-467923-unsplash.jpg") no-repeat center center;
+  background-size: cover;
```
Comment (tawebbcn) [inline students/nico/styles.css:115]: I think that it won't be there. The path is different: "img/...." not the things before 😄

Comment (nicolacaze) [reply students/nico/styles.css:115]: ahah true thanks for this one 😄 !

File: students/nico/index.html (hunk 1 above)
Comment (nicolacaze) [reply students/nico/index.html]: I made a few changes as you suggested.

**Pull request merged**